cmake/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -235,19 +236,45 @@ file(
     "  : ${toolset_version}\n"
 )
 
-if(MSVC)
-  # For Visual Studio C++ flags must not be set in compiler section.
-  # Section <compileflags> should be used.
-  #   * https://github.com/ruslo/hunter/issues/179
-  file(APPEND ${boost_user_jam} "  : \"${boost_compiler}\"\n")
-else()
+if(ANDROID)
   # For Android C++ flags must be part of the compiler section:
   #   * https://github.com/ruslo/hunter/issues/174
-  # For 'sanitize-address' toolchain flags must be part of the compiler section:
-  #   * https://github.com/ruslo/hunter/issues/269
+  # But such approach doesn't work for Visual Studio:
+  #   * https://github.com/ruslo/hunter/issues/179
```

Comment:
I think conflict resolved wrongly here. The last "good" line is `For 'sanitize-address' toolchain flags must be part of the compiler section`